### DISCUSSION TITLE
Improve contrast of tabs with validation errors for higher visibility

### DISCUSSION THREAD

**Discussion by aaronbushnell** [craftcms/cms#11926] [Ideas] [answered]
We've had a couple occasions where people will ask our development team why an entry "won't save". Most of the time it's because there's a validation error, but it's not _entirely_ obvious that's what's happening.

While this isn't a silver bullet, updating the contrast on the invalid tabs might help a bit. I pulled these colors from some other CSS variables:

```
var(--error-color)
var(--red-050)
``` 

![Screen Shot 2022-09-13 at 9 46 43 AM](https://user-images.githubusercontent.com/315202/189918109-5ed2cd21-0e73-4342-a0e4-0e9a32957fa6.png)

Or the following if the background change is too excessive.
![Screen Shot 2022-09-13 at 9 52 49 AM](https://user-images.githubusercontent.com/315202/189919580-abbcaf72-d54b-4d2e-adbb-7f235a8f4643.png)

**Comment by brandonkelly** [comment, marked as answer]
Good idea. Changed for Craft 4.3! (84af6fdd2ac535707ad15bb15cc058715ada1e2b)

<img src="https://user-images.githubusercontent.com/47792/192610491-f103b28c-fb9c-4970-b549-62ad078b93ea.png" width="264" height="88" alt="Two field layout tabs, “Content” and “SEO”, with Content selected. The content tab’s text, warning label, and top selection indicator are all red, indicating that that it contains validation errors.">

<img src="https://user-images.githubusercontent.com/47792/192610546-a87145e8-e6be-4479-971f-0e8903e98cab.png" width="264" height="88" alt="Two field layout tabs, “Content” and “SEO”, with “SEO” selected. The content tab’s text and warning label are red, indicating that it contains validation errors.">

**Reply by aaronbushnell** [reply]
Love it! Thanks, @brandonkelly!
